include/fc/io/fstream.hpp

```diff
@@ -3,16 +3,20 @@
 #include <fc/filesystem.hpp>
 #include <fc/io/iostream.hpp>
 
+#include <fstream>
+
+using namespace std;
+
 namespace fc {
   class path;
   class ofstream : virtual public ostream {
     public:
       enum mode { out, binary };
```

Comment:
I'd recommend remove this definition.

Reply:
@abitmore if I will remove this definition as you suggested then I will have compile errors, couldn't compile without those definitions, or I've missed something maybe and I'm wrong ?

Reply:
Fix the errors by placing ``std::`` in front of the offending function/type definition. Be careful that you're fixing the actual compile error and not changing from a [boost or fc function] to a [standard library function].

Reply:
@jmjatlanta there are placed already std:: but compiler cannot see the definition and that's why I've compile errors, how is it possible to use something without including it ?

Reply:
Oops, sorry. I misunderstood what abitmore was asking. Please ignore my above comment. Post your compile errors somewhere (I recommend http://pastebin.com, as it doesn't mess up formatting), and I'll give you some pointers.

Reply:
@jmjatlanta no worries :-) Thanks for your efforts ! Here is link https://pastebin.com/fd8NgAqJ

Reply:
Yes, it looks like you interpreted Abitmore's comment incorrectly as did I. He's asking you to remove line 14 above. The enum.

Reply:
I mean remove the line `enum mode { out, binary };` to prevent it from being (mis)used.

Your compiler errors seem not related to the change. What have you changed?

Reply:
@jmjatlanta yes you're absolutely right that I've misunderstood @abitmore 's suggestion :-)
@abitmore I've thought that you told about next lines of code which are selected as green in your comment :

#include <fstream>

using namespace std;

Now I've understood your suggestion correctly about line 14